Modules/pyexpat.c

```diff
@@ -1525,6 +1545,11 @@ xmlparse_clear(PyObject *op)
     xmlparseobject *self = xmlparseobject_CAST(op);
     clear_handlers(self, 0);
     Py_CLEAR(self->intern);
+    // NOTE: We cannot call Py_CLEAR(self->parent) prior to calling
+    //       XML_ParserFree(self->itself) or a subparser (created via
+    //       XML_ExternalEntityParserCreate could lose its parent XML_Parser
+    //       while still making use of it internally.
+    //       https://github.com/python/cpython/issues/139400
```

Comment:
```suggestion
    // NOTE: We cannot call Py_CLEAR(self->parent) prior to calling
    //       XML_ParserFree(self->itself) in xmlparse_dealloc() since
    //       otherwise a subparser could lose its parent XML_Parser
    //       while still making use of it internally.
    //       See https://github.com/python/cpython/issues/139400.
```

Since XML_ExternalEntityParserCreate is a long word, we can drop it here. I think it'll be clear enough (and that way, I don't need to update this comment if we were to ever have a way to create subparsers differently in the future).